docs/pr-guide.md

```diff
@@ -0,0 +1,66 @@
+# Protocol Publishing Guide
+
+## Publish a new protocol definition
+1. Fork repo
+2. Choose protocol name and version. Example: `my-cool-protocol`, `0.1`
+3. Create a folder `site/content/protocols/<protocol-name>/<protocol-version>`
+4. Create a protocol definition document in that folder as a markdown file named `readme.md`. Use the following example as a [reference](./protocol-example.md)
+5. Send a Pull Request to the main branch
+
```

Comment:
I believe we need to say something about our CI (here and in  Update an existing protocol definition):
```
6. Wait until all CI checks are complete, and fix the errors if there are any. 
```